custom_components/tuya_local/devices/powerworld_pw040.yaml

```diff
@@ -0,0 +1,478 @@
+name: PW040
+products:
+  - id: 5oc9wmac3bbidekc
+    name: Powerworld PW040 (Vertriebsbezeichnung Michl SMP13)
```

Comment:
The German scattered throughout this config makes it too much effort for me to review fully.